run.py

```diff
@@ -304,6 +324,24 @@ def blind(temp_dir, intent_train_file, workspace_base_file, figure_path,
             print('Generated precision curves for blind set')
         else:
             raise RuntimeError('Failure in plotting curves')
+
+        blind_result_file = test_out_path
+        metrics_args = [sys.executable, INTENT_METRICS_PATH,
+                        '-i', blind_result_file,
+                        '-o', blind_result_file+".metrics.csv",
+                        '--partial_credit_on', str(partial_credit_table is not None)]
+        if subprocess.run(metrics_args).returncode == 0:
+            print('Generated intent metrics')
+        else:
+            raise RuntimeError('Failure in generating intent metrics')
+
+        confusion_args = [sys.executable, INTENT_METRICS_PATH,
```

Comment:
Looks like you have the wrong parameter there!  Maybe CONFUSION _MATRIX _PATH or something?

Reply:
Fixed in commit #51f08b0